src/sagemaker_inference/parameters.py

```diff
@@ -24,3 +24,4 @@
 BIND_TO_PORT_ENV = "SAGEMAKER_BIND_TO_PORT"  # type: str
 SAFE_PORT_RANGE_ENV = "SAGEMAKER_SAFE_PORT_RANGE"  # type: str
 MULTI_MODEL_ENV = "SAGEMAKER_MULTI_MODEL"  # type: str
+VMARGS = "VMARGS"  # type: str
```

Comment:
NIT: Given all other env variables start with `SAGEMAKER_` should we keep it consistent for `VMARGS` as well?

Reply:
Thanks, I think `SAGEMAKER_MODEL_SERVER_VMARGS` makes more sense, to keep it similar to workers/timeout env variables, I've updated the PR accordingly. Let me know what you think @maaquib @davidthomas426